dll/win32/shell32/CDefView.cpp

```diff
@@ -193,6 +193,7 @@ class CDefView :
         virtual HRESULT STDMETHODCALLTYPE GetSpacing(POINT *ppt);
         virtual HRESULT STDMETHODCALLTYPE GetDefaultSpacing(POINT *ppt);
         virtual HRESULT STDMETHODCALLTYPE GetAutoArrange();
+        virtual HRESULT STDMETHODCALLTYPE GetSnapToGrid();
```

Comment:
`GetSnapToGrid` does not belong in the COM interface, the rest of these functions does (Note the `IFolderView methods` above this block)
so there is no need for it to be virtual, returning an `HRESULT` or it being of the calling convention `STDMETHODCALLTYPE`
(However I understand that it returns an `HRESULT` since that is in line with the rest of the `Get` functions.

A better place would be in the `private` block around L118 (if it is used only inside the `CDefView` itself),
or the second public block arround L155

Reply:
Sounds good, I'll get this fixed up tonight and update the PR.